conf/messages.cy

```diff
@@ -69,7 +69,8 @@ external.error.500.list-item2=Too many users on the service
 
 internal.error.500.title=There is a problem with the service
 internal.error.500.heading=Sorry, there is a problem with the service
-internal.error.500.message=Contact the Helpline.
+internal.error.500.line1=Try again later.
```

Comment:
Title (line 70) will need to match heading to include 'Sorry'

Reply:
Good point. I missed that

Reply:
sure thing.. will change